src/application.js

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable guard-for-in */
 import $ from './jquery-mini';
 
-class Application {
+export default class Application {
 
   constructor(options) {
+    this.$$family = 'application';
```

Comment:
Not urgent, but we should check if these tags are actually being used. I've only seen one or two places where any of the $$ constants are being used. 
If not used let's remove. If they are being used, are they something that could be handled with a standard language construct like instanceof?